src/python/pants/init/logging.py

```diff
@@ -42,13 +42,70 @@ def _maybe_configure_extended_logging(logger):
     _configure_requests_debug_logging()
 
 
+def init_rust_logger(level):
+  native = Native()
+  levelno = get_numeric_level(level)
+  print("BL: {}, I want to init logging with level {}".format(os.getpid(), level))
+  native.init_rust_logging(levelno)
+
+
+def setup_logging_to_stderr(python_logger, level):
+  native = Native()
+  levelno = get_numeric_level(level)
+  handler = create_native_stderr_log_handler(levelno, native, stream=sys.stderr)
```

Comment:
Do you want a unicode stream or a bytes string here? If unicode, the idiom we need to use is `sys.stderr.buffer if PY3 else sys.stderr`.

Python 2 uses a bytes interface, whereas Python 3 changes `sys.stderr` to be unicode.

In general, we want to use unicode where ever possible, but in this code base have defaulted to using bytes for `sys.std{out,err}`.

Reply:
The default value for this in python3's `logging.StreamHandler.__init__` is `sys.stderr` which suggests this should be unicode, and we should change callers if needed.

Reply:
Ah yes you're right. This was some of the wonkiest code to fix with Python 3 in the Pants Daemon related logging. See https://github.com/pantsbuild/pants/commit/c5d4ea17fd3105dab80146b263b3869c9f14213c#diff-cde06acc1ffb450493ca1a50ee21b5c5R44.